Source/Core/Core/IOS/SDIO/SDIOSlot0.cpp

```diff
@@ -49,10 +48,10 @@ void SDIOSlot0Device::EventNotify()
 {
   if (!m_event)
     return;
-  // Accessing SConfig variables like this isn't really threadsafe,
-  // but this is how it's done all over the place...
-  if ((SConfig::GetInstance().m_WiiSDCard && m_event->type == EVENT_INSERT) ||
-      (!SConfig::GetInstance().m_WiiSDCard && m_event->type == EVENT_REMOVE))
+
+  const bool sd_card_inserted = Config::Get(Config::MAIN_WII_SD_CARD);
+  if ((sd_card_inserted && m_event->type == EVENT_INSERT) ||
+      (!sd_card_inserted && m_event->type == EVENT_REMOVE))
```

Comment:
It would be cool if we could make this code use callbacks to detect the SD card being inserted/ejected instead of needing to manually call this EventNotify function. In particular, it would make it possible to insert/eject SD cards during emulation on Android. But we can save that for another PR.